corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -228,6 +227,29 @@ def toggle_enabled(request, toggle_or_toggle_name):
     return _toggle_enabled(corehq.toggles, request, toggle_or_toggle_name)
 
 
+@register.simple_tag
+def toggle_js_url(domain, username):
+    return '{url}?username={username}&cachebuster={domain_cb}-{user_cb}'.format(
+        url=reverse('toggles_js', args=[domain]),
+        username=username,
+        domain_cb=toggle_js_domain_cachebuster(domain),
+        user_cb=toggle_js_user_cachebuster(username),
+    )
+
+
+@quickcache(['domain'], timeout=30 * 24 * 60 * 60)
+def toggle_js_domain_cachebuster(domain):
+    # to get fresh cachebusters on the next deploy
+    # change the date below (output from *nix `date` command)
+    #   Thu Mar  3 16:21:30 EST 2016
```

Comment:
We'll want that if we change the format of that file, say, to also include privileges or something. We'll then need all browsers to reload the file. Changing the date in this function changes the quickcache keys, which will result in generating a new value here, which will result in the browser doing a fresh request.